geode-core/src/main/java/org/apache/geode/internal/cache/wan/parallel/ParallelGatewaySenderQueue.java

```diff
@@ -1201,23 +1203,25 @@ protected void addRemovedEvent(PartitionedRegion prQ, int bucketId, Object key)
 
   public void sendQueueRemovalMesssageForDroppedEvent(PartitionedRegion prQ, int bucketId,
```

Comment:
I know you didn't add this method, but can you rename it (from Messsage to Message)?